common.go

```diff
@@ -178,7 +178,7 @@ func dockerVersionCheck() error {
 	}
 	vMin, err := semver.NewVersion(minRequiredDockerVersion)
 	if err != nil {
-		return fmt.Errorf("our bad, sorry... please make an issue.", err)
+		return fmt.Errorf("our bad, sorry... please make an issue, original error: %v", err)
```

Comment:
Can you remove "original error", it's not a different error, it's the only error.